mapflow/mapflow.py

```diff
@@ -1621,31 +1622,33 @@ def calculate_aoi_area_catalog(self) -> None:
             image = self.data_catalog_service.selected_image()
             mosaic = self.data_catalog_service.selected_mosaic()
             if self.dlg.useImageExtentAsAoi.isChecked():
-                if image:
-                    aoi = QgsGeometry().fromWkt(image.footprint)
-                elif mosaic:
-                    aoi = QgsGeometry().fromWkt(mosaic.footprint)
+                if image or mosaic:
```

Comment:
In line 1637 it was necessary to add 'if image or mosaic', here - not necessary, but I thought it may look more consistent (?)